benchmark_scripts/bench.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+DATADIR=$(pwd)
+
+module load mpi
+
+source quick.rc
+
+mkdir -p $DATADIR/benchmark/
+
+location=${1:-benchmarkinputs}
+
+for fileraw in $location/*
+do
+        file=$(basename $fileraw .in)
+        for number in 1 2 4 6 8 12 16
```

Comment:
I would define the list of tasks in a variable at the top of the script so it can be easily edited
```
TASKS_TO_BENCHMARK='1 2 4 6 8 12 16 32 48 64 80 96 112 128'
```